### ISSUE TITLE
What's the difference between Bold browser and Ungoogled Chromium?

### ISSUE THREAD

**Issue by TheEvilSkeleton** [BoldBrowser/bold-browser#2]

**Comment by ghost** [comment]
As it says here 
https://github.com/BoldBrowser/bold-browser/blob/master/docs/building.md
You're building ungoogled-chromium

And here
https://github.com/BoldBrowser/bold-browser#downloads

It says "ungoogled-chromium" in repos and overlays.

What is Bold Browser?

**Comment by PoorPocketsMcNewHold** [comment]
https://en.cryptonomist.ch/2020/06/11/brave-browser-fork-here-comes-braver/

https://decrypt.co/35146/brave-forces-rival-browser-braver-change-name

tl;dr : It was a bloatless Brave Browser fork. Due to copyrights, and diverses attacks from the former to shutdown their fork, they´ve rebranded themselves as Bold Browser, and decided to rebase the project on Ungoogled-chromium. Currently, the public repo is just a fork of ungoogled-chromium with no modifications, but intern work is being done on a private repository, to prepare the first build of BoldBrowser.

**Comment by cromatikap** [comment]
Thanks for this fork, that's awesome. 

Please consider adding an crypto based ad economy without any third-party it would be awesome.

That's to say: 
- user can always opt-in/opt-out seeing textual ads as Brave does already
- Anyone can interact with a smart-contract to add a textual advert limited in characters + link to its website and top-up its ad with Ethereum directly (or BAT, whatever)
- When browser user click on the ad, a part of the token stored in the smart contract is sent automatically to the public address stored in the browser
- the community could grow words blocklists along the way to avoid trashy ads and users could subscribe to it easily in some settings from the browser

And we'll get a trustless "circular economy" working by itself without any third-party to handle it

**Comment by TheEvilSkeleton** [comment]
> https://en.cryptonomist.ch/2020/06/11/brave-browser-fork-here-comes-braver/
> 
> https://decrypt.co/35146/brave-forces-rival-browser-braver-change-name
> 
> tl;dr : It was a bloatless Brave Browser fork. Due to copyrights, and diverses attacks from the former to shutdown their fork, they´ve rebranded themselves as Bold Browser, and decided to rebase the project on Ungoogled-chromium. Currently, the public repo is just a fork of ungoogled-chromium with no modifications, but intern work is being done on a private repository, to prepare the first build of BoldBrowser.

Thank you. This seems to answer my question well.

**Comment by Kellegram** [comment]
> Thanks for this fork, that's awesome.
> 
> Please consider adding an crypto based ad economy without any third-party it would be awesome.
> 
> That's to say:
> 
>     * user can always opt-in/opt-out seeing textual ads as Brave does already
> 
>     * Anyone can interact with a smart-contract to add a textual advert limited in characters + link to its website and top-up its ad with Ethereum directly (or BAT, whatever)
> 
>     * When browser user click on the ad, a part of the token stored in the smart contract is sent automatically to the public address stored in the browser
> 
>     * the community could grow words blocklists along the way to avoid trashy ads and users could subscribe to it easily in some settings from the browser
> 
> 
> And we'll get a trustless "circular economy" working by itself without any third-party to handle it

Lol, why would they ruin their browser just like Brave did?

**Comment by cromatikap** [comment]
> > Thanks for this fork, that's awesome.
> > Please consider adding an crypto based ad economy without any third-party it would be awesome.
> > That's to say:
> > ```
> > * user can always opt-in/opt-out seeing textual ads as Brave does already
> > 
> > * Anyone can interact with a smart-contract to add a textual advert limited in characters + link to its website and top-up its ad with Ethereum directly (or BAT, whatever)
> > 
> > * When browser user click on the ad, a part of the token stored in the smart contract is sent automatically to the public address stored in the browser
> > 
> > * the community could grow words blocklists along the way to avoid trashy ads and users could subscribe to it easily in some settings from the browser
> > ```
> > 
> > 
> > And we'll get a trustless "circular economy" working by itself without any third-party to handle it
> 
> Lol, why would they ruin their browser just like Brave did?

There is a significant difference between what I describe and what Brave do with uphold.

Why do you think this would "ruin" the browser "just like Brave"? 🤔
😄

**Comment by ghost** [comment]
If you want Brave and the features of Brave then use Brave. What is the point of this project to be a better version of Brave if they're just going to do the same things Brave do.

**Comment by cromatikap** [comment]
Brave (and any other browser at this day) does not provide the feature I want then you likely don't understand my proposal.

**Comment by ghost** [comment]
@bidetaggle I think the primary goal of Bold Browser is to be solely a privacy browser that can also do sync and other things mainstream browsers can do. I don't think the crypto/ad stuff you mentioned is really relevant to this project but if you really want those features, you could try filing an issue with Brave Browser and see if they'll implement them 😀

**Comment by Kellegram** [comment]
^ This
What is the point of forking something and making your own only to start re-adding the same shit just coz some random folk want their feature back. How can you differentiate yourself properly from the original if you start replicating it, especially the unethical parts of it. 

Brave is missing a feature you want? Request them, or some other fork that is trying to be more like Brave.

**Comment by cromatikap** [comment]
# @gottabeleGIT 

> @bidetaggle I think the primary goal of Bold Browser is to be solely a privacy browser that can also do sync and other things mainstream browsers can do. I don't think the crypto/ad stuff you mentioned is really relevant to this project but if you really want those features, ...

Privacy is an essential right and not a tool exclusively designed to watch porn but to enforce individual user rights on the internet such as censorship, freedom of speech, freedom of sharing information etc... without being spied on.
Cryptography is an unavoidable field when it comes to privacy and a technology made of cryptography and decentralization addresses very well to privacy and censorship issues on the internet, depending on its design and usage. 

It's pretty hard to understand how this wouldn't be relevant to this project, it's likely that you don't understand my proposal so if one of you have actual questions I would be happy to answer. 

It's obvious, tho, that you won't really make something because everybody in here agree to brutally reject a potential contributor on a subject they manifestly don't know very well, for obscure reasons, on 1 out of total 3 public issues 🙃

-------

> ... you could try filing an issue with Brave Browser and see if they'll implement them 😀

Brave has already this built-in system to fund them-self fully controlled by a third-party (https://uphold.com/).

You can see [in this comment of me](https://community.brave.com/t/how-do-i-pay-to-advertise-to-brave-users/95269/2) that the process to submit an add is totally close and obscure to the regular users (and I never succeeded to do it).

I tried all of the extends of their system, I made a "Brave Verified Creator" and received BAT for having opt-in ads while browsing. 
It's all in the same fashion, centralized, monitored and regulated by Brave and Uphold with no transparency on the user data and donations but KYC and unreasonable waiting time on the user side. 
You never get your BAT from a neutral smart-contract but you must login to uphold which basically disable all the blockchain/privacy idea

In other words, they're faking it and they'll definitely never be interested in a real [decentralized user-owned competitor](https://github.com/BoldBrowser/bold-browser/issues/2#issuecomment-673469791) to their system, I don't need to post anything there to know it.

-------

# @Kellegram 

> What is the point of forking something and making your own only to start re-adding the same shit just coz some random folk want their feature back.

Isn't it what you are doing right now ? Adding the same _"shit"_ like you could just install a firefox/chromium and [and a few plugins](https://www.privacytools.io/browsers/#addons) instead of forking an entire bare browser. 
Are we sure to do better than this solution in an up-to-date and out of the box browser? 

If this fork appears someday on the first page of [alternativeto.net](https://alternativeto.net/software/iridium/), that would be great, but, once again I'm speaking about something that doesn't exist nowhere yet so [it]((https://github.com/BoldBrowser/bold-browser/issues/2#issuecomment-673469791)) can be ~~the same~~ __shit__ ~~of something~~, you got it? 😅

**Comment by PoorPocketsMcNewHold** [comment]
> [**@bidetaggle**](https://github.com/bidetaggle) commented on [Aug 19, 2020, 3:47 PM GMT+2](https://github.com/BoldBrowser/bold-browser/issues/2#issuecomment-676386419):
> 
> # [@gottabeleGIT](https://github.com/gottabeleGIT)
> 
> [@bidetaggle](https://github.com/bidetaggle) I think the primary goal of Bold Browser is to be solely a privacy browser that can also do sync and other things mainstream browsers can do. I don't think the crypto/ad stuff you mentioned is really relevant to this project but if you really want those features, ...
> 
> Privacy is an essential right and not a tool exclusively designed to watch porn but to enforce individual user rights on the internet such as censorship, freedom of speech, freedom of sharing information etc... without being spied on.  
> Cryptography is an unavoidable field when it comes to privacy and a technology made of cryptography and decentralization addresses very well to privacy and censorship issues on the internet, depending on its design and usage.
> 
> It's pretty hard to understand how this wouldn't be relevant to this project, it's likely that you don't understand my proposal so if one of you have actual questions I would be happy to answer.

By cryptography, you are just requesting a crypto-economy which isn´t needed. The previous instance that we shall not name, actually removed all of it from Brave.

Bold browser isn´t made to be a profitable project. 


> # [@Kellegram](https://github.com/Kellegram)
> 
> What is the point of forking something and making your own only to start re-adding the same shit just coz some random folk want their feature back.
> 
> Isn't it what you are doing right now ? Adding the same _"shit"_ like you could just install a firefox/chromium and [and a few plugins](https://www.privacytools.io/browsers/#addons) instead of forking an entire bare browser.  
> Are we sure to do better than this solution in an up-to-date and out of the box browser?
> 
> If this fork appears someday on the first page of [alternativeto.net](https://alternativeto.net/software/iridium/), that would be great, but, once again I'm speaking about something that doesn't exist nowhere yet so it can be ~the same~ **shit** ~of something~, you got it?
> 
> ![sweat_smile](https://github.githubassets.com/images/icons/emoji/unicode/1f605.png)

Outside the Tor Browser, which is indeed limited by the TOR protocol, they are no out of the box, entirely private browsers. The only one I can found, is [Librewolf](Librewolf-community.gitlab.io/), which is much more just a Firefox config.
Bold Browser is aiming to be the more privacy-friendly to the user, throughout the web browser. It doesn’t try to aim a balance between functionality and privacy, this isn’t possible. It will try to aim to the more private possible.

**Comment by cromatikap** [comment]
> Bold browser isn´t made to be a profitable project.

As well as my feature proposal.

I wish the best for y'all
